### DISCUSSION TITLE
Deployed contracts

### DISCUSSION THREAD

**Discussion by isha289** [ConsenSys-archive/truffle#6224] [General Q&A] [answered]
Hi guys,
Since the truffle is now shutting down what about the contracts which are already deployed? Do we need to migrate them to new framework or it should work as it is?

**Comment by MadelineMurray** [comment, marked as answer]
hi @isha289 -yes existing deployed contracts will operate as they are. Additionally, there’s no reason why Truffle will suddenly stop working if you want to keep using it in the interim while migrating to a new framework (although given Truffle is no longer being maintained migrating is important).

**Reply by isha289** [reply]
Thanks a lot.
I am planning on migration but I dont have to immediately start it right. For a while truffle is good!
